commit/trk2dictionary/trk2dictionary.pyx

```diff
@@ -8,19 +8,22 @@ import nibabel
 from os.path import join, exists
 from os import makedirs, remove
 import time
+import os.path
+import array
 
 
 # Interface to actual C code
 cdef extern from "trk2dictionary_c.cpp":
     int trk2dictionary(
-        char* strTRKfilename, int Nx, int Ny, int Nz, float Px, float Py, float Pz, int n_count, int n_scalars, int n_properties, float fiber_shiftX, float fiber_shiftY, float fiber_shiftZ, int points_to_skip, float min_seg_len,
+        char* filename_tractogram, int data_offset, int Nx, int Ny, int Nz, float Px, float Py, float Pz, int n_count, int n_scalars, 
+        int n_properties, float fiber_shiftX, float fiber_shiftY, float fiber_shiftZ, int points_to_skip, float min_seg_len,
         float* ptrPEAKS, int Np, float vf_THR, int ECix, int ECiy, int ECiz,
         float* _ptrMASK, float* ptrTDI, char* path_out, int c, double* ptrAFFINE,
-        int nBlurRadii, double blurSigma, double* ptrBlurRadii, int* ptrBlurSamples, double* ptrBlurWeights
+        int nBlurRadii, double blurSigma, double* ptrBlurRadii, int* ptrBlurSamples, double* ptrBlurWeights,  float* ptrArrayInvM
     ) nogil
 
 
-cpdef run( filename_trk, path_out, filename_peaks = None, filename_mask = None, do_intersect = True,
+cpdef run( filename_tractogram, path_out, TCK_ref_image = None, filename_peaks = None, filename_mask = None, do_intersect = True,
```

Comment:
filename_tractogram is not compatible with previous scripts. 
Documentation and tutorials have different variable name

Reply:
Change `filename_tractogram = None, path_out = None` and add as last parameter `filename_trk = None`.

Then, check that [`path_out` is not None]  AND  [at least one of `filename_tractogram` or `filename_tractogram` is not None]. If the previous is not satisfy, rise and error. 

If `filename_trk`is used, show (print) a warning, this parameter will be deprecated. 

If `filename_tractogram` and `filename_trk` are used, show a message like:  filename_tractogram will be used, filename_trk will not be considered"